doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -390,7 +391,7 @@ nRF Machine Learning (Edge Impulse)
 Thingy:53: Matter weather station
 ---------------------------------
 
-|no_changes_yet_note|
+* Updated the :ref:`matter_weather_station_app` sample to use the code-driven approach for the Identify cluster implementation instead of the zap-driven approach.
```

Comment:
Maybe you do not need to link to the the app (not sample) as this section is all about the Matter weather station.